### PR TITLE
Added a deepgramAudioSummary() Function using PHP

### PR DESCRIPTION
**Pull request by neelshah2409**
A Php Cloud Function for audio summary using [Deepgram API](https://developers.deepgram.com/documentation/).

### CODE REVIEW

**Comment by neelshah2409** [comment]
Made a PR please review it.
@Meldiron @stnguyen90

**Comment by PineappleIOnic** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Comment by neelshah2409** [comment]
Made a PR please review it. @Meldiron @stnguyen90

File: php/deepgram-Audio-Summary/Readme.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+_Example function payload:_
+
+```json
+{
+    {
+
+        "fileUrl":"https://rawcdn.githack.com/deepgram-devs/summary-audio/62fc7769d6e2bf38e420ee5224060922af4546f7/deepgram.mp3"
+
+    }
+}
```
Comment (stnguyen90) [inline php/deepgram-Audio-Summary/Readme.md:15]: This payload is not valid JSON

File: php/deepgram-Audio-Summary/Readme.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+**DEEPGRAM_API_KEY** - Your Deepgram API key.
+
+## 🚀 Deployment
+
+1. Clone this repository, and enter this function folder:
+
+```
+$ git clone https://github.com/open-runtimes/examples.git && cd examples
+$ cd php/deepgram-audio-summary
```
Comment (stnguyen90) [inline php/deepgram-Audio-Summary/Readme.md:42]: Please update the folder name to match this.

File: php/deepgram-Audio-Summary/Readme.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+```
+docker run --rm --interactive --tty --volume $PWD:/usr/code openruntimes/php:v2-8.1 sh /usr/local/src/build.sh
+```
+
+As a result, a `code.tar.gz` file will be generated.
+
+3. Start the Open Runtime:
+
+```
+docker run -p 3000:3000 -e INTERNAL_RUNTIME_KEY=secret-key -e INTERNAL_RUNTIME_ENTRYPOINT=main.py --rm --interactive --tty --volume $PWD/code.tar.gz:/tmp/code.tar.gz:ro openruntimes/php:v2-8.1 sh /usr/local/src/start.sh
```
Comment (stnguyen90) [inline php/deepgram-Audio-Summary/Readme.md:56]: This is the incorrect command to run this runtime container.

File: php/deepgram-Audio-Summary/Readme.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+```
+
+> Make sure to replace `YOUR_API_KEY` without your key.
+
+Your function is now listening on port `3000`, and you can execute it by sending `POST` request with appropriate authorization headers. To learn more about runtime, you can visit PHP runtime [README](https://github.com/open-runtimes/open-runtimes/tree/main/runtimes/php-8.1).
+
+## 📝 Notes
+
+- This function is designed for use with Appwrite Cloud Functions. You can learn more about it in [Appwrite docs](https://appwrite.io/docs/functions).
+- This example is compatible with PHP 2.8.1. Other versions may work but are not guaranteed to work as they haven't been tested.
```
Comment (stnguyen90) [inline php/deepgram-Audio-Summary/Readme.md:66]: Please use the correct version number

```suggestion
- This example is compatible with PHP 8.1. Other versions may work but are not guaranteed to work as they haven't been tested.
```

File: php/deepgram-Audio-Summary/index.php [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+        "Authorization: Bearer " + $secretKey
+    ));
+ 
+    $response = curl_exec($ch);
+    curl_close($ch);
+ 
+    return $response;
+}
+ 
+?>
```
Comment (stnguyen90) [inline php/deepgram-Audio-Summary/index.php:20]: This is not the correct way to implement a PHP Open Runtimes Function. Please see how the other examples are structured and be sure to test the function like documented in the readme.

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

**Review by Meldiron — APPROVED** [review]

**Comment by Meldiron** [comment]
Thanks for the amazing PR 😇 I have added small changes to fix function structure and readme.
Merging the PR now 🥳

**Pull request merged**